dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsRowMapper.java

```diff
@@ -93,6 +96,7 @@ public SubSnpCoreFields mapRow(ResultSet resultSet, int rowNum) throws SQLExcept
                 resultSet.getLong(CONTIG_START_COLUMN),
                 resultSet.getLong(CONTIG_END_COLUMN),
                 resultSet.getInt(CONTIG_ORIENTATION_COLUMN),
+                LocationType.fromInteger(resultSet.getInt(LOC_TYPE_COLUMN)),
```

Comment:
Call `LocationType.fromInteger` inside the constructor to ensure the object is correctly constructed without intervention from the reader. This is how the `Orientation` enums are constructed too.